src/overlays/actors/ovl_En_Sda/z_en_sda.c

```diff
@@ -21,17 +30,371 @@ const ActorInit En_Sda_InitVars = {
     (ActorFunc)EnSda_Update,
     (ActorFunc)EnSda_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sda/EnSda_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Sda/EnSda_Destroy.s")
+static Vec3f D_80AFA0D0 = { 0.0f, 0.0f, 0.0f };
+
+static s16 D_80AFA0DC[] = {
+    1, 2, 3, 3, 2, 1,
+};
+
+static s16 D_80AFA0E8[] = {
+    2, 3, 4, 4, 4, 3, 2, 0,
+};
+
+static s16 D_80AFA0F8[] = {
+    2, 3, 4, 4, 4, 4, 3, 2,
+};
+
+static s16 D_80AFA108[] = {
+    2, 4, 5, 5, 6, 6, 6, 6, 5, 5, 4, 2,
+};
+
+static s16 D_80AFA120[] = {
+    2, 4, 5, 6, 7, 8, 8, 8, 8, 7, 6, 5, 4, 2,
+};
+
+static s16 D_80AFA13C[] = {
+    1, -1, 1, 1, 3, 4, 1, 6, 7, 2, 9, 10, 2, 12, 13, 0,
+};
+
+static u8 D_80AFA15C[] = {
+    2, 2, 2, 3, 3, 3, 3, 3, 3, 0, 0, 0, 0, 0, 0, 3,
+};
+
+static s8 D_80AFA16C[] = {
+    2, 9, 10, 11, 12, 13, 14, 0, 15, -1, 3, 4, 5, 6, 7, 8, -1, 1, 0, 0,
+};
+
+static Vec3f D_80AFA180[] = {
+    { -1.0f, 2.0f, -0.2f }, { 0.0f, 2.0f, -0.5f },   { 1.0f, 2.0f, -0.2f },   { -2.0f, 1.0f, -0.5f },
+    { -1.0f, 1.0f, -0.2f }, { 0.0f, 1.0f, -0.2f },   { 1.0f, 1.0f, -0.2f },   { 2.0f, 1.0f, -0.5f },
+    { -2.0f, 0.0f, -0.5f }, { -1.0f, 0.0f, -0.2f },  { 0.0f, 0.0f, 0.0f },    { 1.0f, 0.0f, -0.2f },
+    { 2.0f, 0.0f, -0.5f },  { -2.0f, -1.0f, -0.5f }, { -1.0f, -1.0f, -0.2f }, { 0.0f, -1.0f, -0.1f },
+    { 1.0f, -1.0f, -0.2f }, { 2.0f, -1.0f, -0.5f },  { -1.0f, -2.0f, -0.2f }, { 0.0f, -2.0f, -0.2f },
+    { 1.0f, -2.0f, -0.2f }, { 0.0f, -3.0f, -0.5f },
+};
+
+// Unused, identical to D_80AFA180
+static Vec3f D_80AFA288[] = {
+    { -1.0f, 2.0f, -0.2f }, { 0.0f, 2.0f, -0.5f },   { 1.0f, 2.0f, -0.2f },   { -2.0f, 1.0f, -0.5f },
+    { -1.0f, 1.0f, -0.2f }, { 0.0f, 1.0f, -0.2f },   { 1.0f, 1.0f, -0.2f },   { 2.0f, 1.0f, -0.5f },
+    { -2.0f, 0.0f, -0.5f }, { -1.0f, 0.0f, -0.2f },  { 0.0f, 0.0f, 0.0f },    { 1.0f, 0.0f, -0.2f },
+    { 2.0f, 0.0f, -0.5f },  { -2.0f, -1.0f, -0.5f }, { -1.0f, -1.0f, -0.2f }, { 0.0f, -1.0f, -0.1f },
+    { 1.0f, -1.0f, -0.2f }, { 2.0f, -1.0f, -0.5f },  { -1.0f, -2.0f, -0.2f }, { 0.0f, -2.0f, -0.2f },
+    { 1.0f, -2.0f, -0.2f }, { 0.0f, -3.0f, -0.5f },
+};
+
+static u32 D_80AFA390[] = { 0, 0 };
```

Comment:
No, those are u32s. Definitely going to shift if they're removed.

Reply:
this should be padding to align Vtx to 8

Reply:
er wait i guess 0 is fine

Reply:
Since this is `.data` it probably won't align enough to match here, but a single 0 would work yeah.
It's a bit weird either way but it could have been anything so who knows